diff --git a/node_modules/react-native-screens/RNScreens.podspec b/node_modules/react-native-screens/RNScreens.podspec
index da0170e..9371e8a 100644
--- a/node_modules/react-native-screens/RNScreens.podspec
+++ b/node_modules/react-native-screens/RNScreens.podspec
@@ -65,6 +65,14 @@ Pod::Spec.new do |s|
   s.source_files = source_files
   s.requires_arc = true
 
+  s.pod_target_xcconfig = {
+    'HEADER_SEARCH_PATHS' => [
+      '"$(PODS_TARGET_SRCROOT)/ios/utils"',
+      '"$(PODS_TARGET_SRCROOT)/cpp"',
+      '"$(PODS_ROOT)/Headers/Public/React-jsi"',
+    ].join(' ')
+  }
+
   if defined?(install_modules_dependencies()) != nil
     install_modules_dependencies(s)
     # Add missing dependencies, that were not included in install_modules_dependencies
